pachyderm/values.yaml

```diff
@@ -1,116 +1,256 @@
+      region: ""
+      # retries sets the number of retries for object storage
+      # requests.  It is analogous to the --retries argument to
+      # pachctl deploy.
+      retries: 10
+      # reverse reverses object storage paths.  It is analogous to the
+      # --reverse argument to pachctl deploy.
+      reverse: true
+      # secret sets the Amazon secret access key to use.  Together with id
+      # and token, it implements the functionality of the
+      # --credentials argument to pachctl deploy.
+      secret: ""
+      # timeout sets the timeout for object storage requests.  It is
+      # analogous to the --timeout argument to pachctl deploy.
+      timeout: "5m"
+      # token optionally sets the Amazon token to use.  Together with
+      # id and secret, it implements the functionality of the
+      # --credentials argument to pachctl deploy.
+      token: ""
+      # uploadACL sets the upload ACL for object storage uploads.  It
+      # is analogous to the --upload-acl argument to pachctl deploy.
+      uploadACL: "bucket-owner-full-control"
+      vault:
+        address: ""
+        role: ""
+        token: ""
+    # backend configures the storage backend to use.  It must be one
+    # of GOOGLE, AMAZON, MINIO, MICROSOFT or LOCAL.
+    backend: ""
+    google:
+      bucket: ""
+      # cred is a GCP service account private key, in object (JSON or
+      # YAML) form, e.g.:
+      #
+      # {
+      #  "type": "service_account",
+      #  "project_id": "…",
+      #  "private_key_id": "…",
+      #  "private_key": "-----BEGIN PRIVATE KEY-----\n…\n-----END PRIVATE KEY-----\n",
+      #  "client_email": "…@….iam.gserviceaccount.com",
+      #  "client_id": "…",
+      #  "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+      #  "token_uri": "https://oauth2.googleapis.com/token",
+      #  "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+      #  "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/…%40….iam.gserviceaccount.com"
+      # }
+      cred:
+        type: "service_account"
+        project_id: ""
+        private_key_id: ""
+        private_key: ""
+        client_email: ""
+        client_id: ""
+        auth_uri: "https://accounts.google.com/o/oauth2/auth"
+        token_uri: "https://oauth2.googleapis.com/token"
+        auth_provider_x509_cert_url: "https://www.googleapis.com/oauth2/v1/certs"
+        client_x509_cert_url: ""
+      serviceAccountName: "" # For Workload Identity access to the bucket
+    local:
+      # hostPath indicates the path on the host where the PFS metadata
+      # will be stored.  It must end in /.  It is analogous to the
+      # --host-path argument to pachctl deploy.
+      hostPath: "/var/pachyderm/"
     microsoft:
-      microsoftContainer: ""
-      microsoftID: ""
-      microsoftSecret: ""
+      container: ""
+      id: ""
+      secret: ""
     minio:
-      minioBucket: ""
```

Comment:
Agree with the name change here, but since there's significant changes here can you add tests for the various object store options to make sure they are wired through correctly?